### PR TITLE
Image widget PR as breaking

### PR DESCRIPTION
**Pull request by sneridagh**
----
📚 Documentation preview 📚: https://volto--6125.org.readthedocs.build/

### CODE REVIEW

File: docs/source/upgrade-guide/index.md [hunk 1]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
 
 The documentation has been updated as well to reflect this change.
 
+
+### New image widget component
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:344]:
```suggestion
### New image upload widget component
```

File: docs/source/upgrade-guide/index.md [hunk 2]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
...
+
+The Image Block edit component had the image upload widget component integrated in it.
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:346]:
```suggestion
Previously the image upload widget component was integrated into the image block edit component.
```

File: docs/source/upgrade-guide/index.md [hunk 3]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
...
+Now the component has been extracted to its own component for reusability in other blocks.
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:347]:
```suggestion
Now the image upload widget component is its own component, and you can reuse it in other blocks.
```

File: docs/source/upgrade-guide/index.md [hunk 4]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
...
+If you shadowed the Image Block edit component, make sure it continues working as intended or update it to use the new image widget.
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:348]:
```suggestion

If you shadow the image block edit component, make sure it continues to work as you expect, or update it to use the new image upload widget component.
```

File: docs/source/upgrade-guide/index.md [hunk 5]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
...
+The user experience of the image upload has changed too.
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:349]:
```suggestion

The new image upload widget component's user experience also changed.
```

File: docs/source/upgrade-guide/index.md [hunk 6]
```diff
@@ -340,6 +340,16 @@ The following table lists the old and new Makefile commands and the new commands
...
+It has been simplified and the URL form is now behind a button, so it's better suited for small screens and potential small containers like when the widget has to be in the grid block elements.
```
Comment (stevepiercy) [inline docs/source/upgrade-guide/index.md:350]:
```suggestion
The input field is now a row of buttons.
The input field's placeholder text was moved above the buttons.
Together these changes improve usability both on small screens and in small containers, such as when the widget is in grid block elements.
```

File: packages/volto/news/5607.breaking [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Added `ImageWidget` with upload, drop, external, and inline capabilities. @sneridagh @ichim-david @dobri1408
```
Comment (stevepiercy) [inline packages/volto/news/5607.breaking:1]:
```suggestion
Added `ImageWidget` component with upload, drop, external, and inline capabilities. @sneridagh @ichim-david @dobri1408
```

File: packages/volto/news/5607.breaking [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+
+Breaking:
+
+The user experience of the image upload has changed.
+The image edit component has been splitted up and now it's using the new image upload widget.
```
Comment (stevepiercy) [inline packages/volto/news/5607.breaking:6]:
```suggestion
The input field is now a row of buttons.
The input field's placeholder text was moved above the buttons.
Together these changes improve usability both on small screens and in small containers, such as when the widget is in grid block elements.
```

File: packages/volto/news/5607.breaking [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+If you shadowed this component, make sure it continues working as intended or update to the new image widget.
```
Comment (stevepiercy) [inline packages/volto/news/5607.breaking:7]:
```suggestion
If you shadow the image block edit component, make sure it continues to work as you expect, or update it to use the new image upload widget component.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by sneridagh** [comment]
@stevepiercy 🙇🏻

**Pull request merged**